### ISSUE TITLE
Can't create custom fields with name "type"

### ISSUE THREAD

**Issue by pierreozoux** [zammad/zammad#3578]
### Infos:

* Used Zammad version: 3.6.0
* Installation method (source, package, ..): docker

### Expected behavior:

* As a user I shouldn't be able to bork my install by adding a custom field to a user object.

### Actual behavior:

* If you use the custom field name type, the server crashes

### Steps to reproduce the behavior:

* Add custom field in user object as the screenshot attached
* Add this to a user
* Get the following 500:

```
StatusCode: 500

{"error":"

The single-table inheritance mechanism failed to locate the subclass:
'Secteur public'. This error is raised because the column 'type' is reserved for storing the class in case of inheritance. Please rename this column if you didn't intend it to be used for storing the inheritance class or overwrite User.inheritance_column to use another column for that information."}
```

Yes I'm sure this is a bug and no feature request or a general question.

<img width="684" alt="Screenshot 2021-05-28 at 15 49 57" src="https://user-images.githubusercontent.com/1863453/119994737-28e18000-bfcd-11eb-873b-f66cd444faec.png">

**Comment by MrGeneration** [comment]
Please do everyone the favor to be a little bit more precise on the steps to reproduce.

**Comment by pierreozoux** [comment]
Sorry, I didn't understand, could you reproduce or not?

**Comment by mantas** [comment]
Yes we did. Eventually.

**Comment by dominikklein** [comment]
Closed with 0fe81f4e0b3b6f01582ffa93a8e74953cc77668a
